diagnostic.md

```diff
@@ -157,7 +176,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne will be to the right,
+div 2,3 Four will be to the left in arranged in a stack like column and
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).